cluster/clean.sh

```diff
@@ -21,25 +27,49 @@ if [ -f "${OPERATOR_MANIFEST}" ]; then
 fi
 
 # Everything should be deleted by now, but just to be sure
-namespaces=(default kube-system)
-for i in ${namespaces[@]}; do
-    _kubectl -n ${i} delete deployment -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete services -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete secrets -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete configmaps -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete pvc -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete pods -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete rolebinding -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete roles -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
-    _kubectl -n ${i} delete serviceaccounts -l 'operator.cdi.kubevirt.io' -l 'cdi.kubevirt.io'
+for n in ${NAMESPACES[@]}; do
+  for label in ${labels[@]}; do
+    _kubectl -n ${n} delete deployment -l ${lael}
```

Comment:
you have a typo, should be label not lael

Reply:
Done, thanks!